ATSQuadroStrategyBase/NinjaTrader 8/bin/Custom/Strategies/AlgoSystemBase.cs

```diff
@@ -2366,7 +2366,7 @@ public virtual StrategyTradeWorkFlowState ProcessWorkFlow(StrategyTradeWorkFlowS
                     TradeWorkFlowOnMarketDataDisable();
                     return ProcessWorkFlow(StrategyTradeWorkFlowState.GoLongSubmitOrder);
                 case StrategyTradeWorkFlowState.GoLongSubmitOrder:
-                    if (IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode)

```

Comment:
This is required -  IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode  use the same paths a lot of the time-